package.json

```diff
@@ -1,15 +1,31 @@
 {
```

Comment:
@jaypaik Let me know if we need to adjust any metadata, thanks!

Reply:
lgtm!